docs/sources/installation/helm/monitor-and-alert/with-grafana-cloud.md

```diff
@@ -0,0 +1,100 @@
+---
+title: Configure Monitoring and Alerting of Loki Using Grafana Cloud
+menuTitle: Monitor Loki with Grafana Cloud
+description: setup monitoring and alerts for Loki using Grafana Cloud
+aliases:
+  - /docs/installation/helm/monitoring/with-grafana-cloud
+weight: 100
+keywords:
+  - monitoring
+  - alert
+  - alerting
+  - grafana cloud
+---
+
+# Configure Monitoring and Alerting of Loki Using Grafana Cloud
+
+This guide will walk you through using Grafana Cloud to monitor a Loki installation that is installed with the Helm chart. This approach leverages many of the chart's _self monitoring_ features, but instead of sending logs back to Loki itself, it sends them to a Grafana Cloud Logs instance. This approach also does not require the installation of the Prometheus Operator and instead sends metrics to a Grafana Cloud Metrics instance. Using Grafana Cloud to monitor Loki as the added benefit of being able to troubleshoot problems with Loki when the Helm installed Loki is down, as the logs will still be available in the Grafana Cloud Logs instance.
+
+**Before you begin:**
+
+- Helm 3 or above. See [Installing Helm](https://helm.sh/docs/intro/install/).
+- A Grafana Cloud account and instance
```

Comment:
(nit) Isn't it called a stack instead of an instance? https://grafana.com/docs/grafana-cloud/account-management/cloud-portal/#your-grafana-cloud-stack

Reply:
This would make sense if that stack includes Grafana Cloud Metrics (since line 17 says this will send metrics to a Grafana Cloud Metrics instance).  Otherwise shouldn't we also list GFM as a prerequisite?

Reply:
I will change to stack, as metrics is required.